### PR TITLE
Recent changes branch project

### PR DESCRIPTION
**Pull request by schoolsofafrika**
Page titles - Add Icon
Image tags - Add alt in the image tag
Text headings - (No Issues Found)
Color contrast (No Issues Found)
Resize (No Issues Found)
Interaction (No Issues Found)
Moving content (No Issues Found)
Multimedia (No Issues Found)
The basic structure of the page (No Issues Found)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -174,6 +174,24 @@ nav {
   border-radius: 10px;
 }
 
+.section-odd .image {
+  transition: 1s;
+}
+
+.section-even .image {
```
Comment (koachcode) [inline style.css:181]: - Good job on your accessibility report, however, it lacks some report requirements listed below:
  
 **No report on**
     - Page title [How to check for Page titles](https://www.w3.org/WAI/test-evaluate/preliminary/#title)
     - image text alternatives `(alt text)` [How to check Image text Alt](https://www.w3.org/WAI/test-evaluate/preliminary/#images)
    - text headings [How to check text heading](https://www.w3.org/WAI/test-evaluate/preliminary/#headings)

**The report  should include a statement pertaining to every point checked:**
- If everything was ok,  make a statement about that (e.g. "Page titles - no issues found").
- If issues were found, Your report should have a  link to the corresponding line of code, in the Pull Request changes

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @schoolsofafrika, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- Good communication
- No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by schoolsofafrika** [comment]
Please I am not clear, what do you mean by " Your report should have a link
to the corresponding line of code, in the Pull Request changes"?
Do I have to attach any link to it?

**Comment by koachcode** [comment]
Hi @schoolsofafrika , What it means is, if the issues were found, you should include a link or a reference to where it is in your code( maybe like a link number). If you still find it confusing please fill free to reach out to me.

**Pull request merged**